build-system/server.js

```diff
@@ -551,6 +555,18 @@ app.get('/dist/rtv/99*/*.js', function(req, res, next) {
   }).catch(next);
 });
 
+app.get(['/dist/rtv/*/v0/amp-ad-0.1.js', '/dist/rtv/*/v0/amp-video-0.1.js'], function(req, res, next) {
```

Comment:
I decide not to remove `/dist` because we will then need it back to get the correct file path.